server_utils/origin.go

```diff
@@ -750,6 +753,29 @@ from S3 service URL. In this configuration, objects can be accessed at /federati
 	return originExports, nil
 }
 
+// Generate a test auth token for checking the sentinel location
+func generateFileTestScitoken(resourceScope string) (string, error) {
+	issuerUrl := param.Server_ExternalWebUrl.GetString()
+	if issuerUrl == "" { // if both are empty, then error
+		return "", errors.New("failed to create token: invalid iss, Server_ExternalWebUrl is empty")
```

Comment:
The error message here is a little Mountain Dewey for the average user. How about something like "failed to create a sentinel test token because a required configuration 'Server.ExternalWebUrl" is unset." 

One thing to note here is the formatting of `Server.ExternalWebUrl`, which is our user-facing formatting because it translates more cleanly into the yaml configurations -- the underscore is an internal representation.